Added support for multi-word seeds

CapacitorSet opened the pull request.

CapacitorSet committed:
Added support for multi-word seeds

neslinesli93 commented:
Why bother to add support for multi word seeds, when the code simply takes the last word and ignores the rest?
With the current implementation, if you supply a sentence it takes the first word. How is this PR an improvement?

CapacitorSet commented:
@neslinesli93 I think this opens up interesting possibilities for the user. It means they can supply a starting sentence, and the bot _appears_ to complete it.

Compare the following exchanges:

> **User**: /chobot I think that
> **Old bot**: I x y z.
> 
> **User**: /chobot I think that
> **New bot**: I think that x y z.

While the sentence generation algorithm doesn't change, the user will have an enhanced experience, because they have more control over what the bot says - thus leading to more opportunities for funny exchanges.

Commit:
Small fixes

When using multi-word seeds, the bot no longer takes its own command into account. Also, when doing random replies, it will seed with the first word only.

blackdev1l commented:
looks good for me :+1: merging
